governance/roles.md

```diff
@@ -173,6 +173,18 @@ One or more of the following:
 Any member who meets the above qualifications is encouraged to add themselves
 to the [list of people in this role](/.github/settings.yml).
 
+### Meeting Scribe(s)
+
+Each meeting must have at a minimal one scribe and preferable with two.
+
+Scribe is expected to perform the following:
+* Review meeting Agenda with facilitator before start of meeting
+* Summarize discussions and capture into meeting notes (not transcribe verbatim)
+* Clarify and annotate actions as discussion progresses with assignee and due dates
+* Partner with Facilitator to capture major occurring items in the meeting
```

Comment:
Hmm, it seems a bit unclear how this should happen. Also, if we include meeting facilitator, we should add a bullet there to work with scribe as well

Reply:
Concur - perhaps coordination of agenda with facilitator via slack #sig-security thread?  Maybe have the facilitator start the thread to request the scribe?

Reply:
I think the faciliator should start the thread to the scribe.  As for the scribe session, changing the first bullet to "Plan to coordinate/review meeting Agenda  with facilitator via Slack before start of meeting"?  I thought about putting "30mins beforehand", but might be best to leave it to Facilitator and Scribe to coordinate instead.

Reply:
concur that sounds good.  @lumjjb ?

Reply:
Yes that sounds good. Let's make that change and I think this should be good to merge

Reply:
Great, updated!

Reply:
Updated, please review and merge! ;)